packages/theme/src/colors/alert.ts

```diff
@@ -76,6 +76,18 @@ export const informative = {
   informative50,
 }
 
+export type SuccessColor = keyof typeof success
+export type WarningColor = keyof typeof warning
+export type CriticalColor = keyof typeof critical
+export type InformativeColor = keyof typeof informative
+export type AlertColor = SuccessColor | WarningColor | CriticalColor | InformativeColor
```

Comment:
Do we need to export these types?

Reply:
Yup, it took a lot of time to get color type and I think this is the best solution to get color type